test/unit/user_test.rb

```diff
@@ -93,6 +93,91 @@ class UserTest < ActiveSupport::TestCase
     end
   end
 
+  test 'send_daily_triage! sends email after preferred email_time_of_day' do
+    user  = users(:morning_person)
+    repo  = repos(:rails_rails)
+
+    issue = repo.issues.new
+    issue.title           = "Foo Bar"
+    issue.url             = "http://schneems.com"
+    issue.last_touched_at = 2.days.ago
+    issue.state           = 'open'
+    issue.html_url        = "http://schneems.com"
+    issue.number          = (repo.issues.last.try(:number) || 0) + 1
+    issue.save
```

Comment:
Thanks @MonikaMahanthappa. I'll take a look to see how it could be simplified. I'm not sure if I will run into issues where some attributes have to be set before they are saved.
I duplicated the existing tests on send_daily_triage! which have this setup.